### ISSUE TITLE
lmfdb_page function in Sage for an Number field doesn't work now

### ISSUE THREAD

**Issue by alexjbest** [LMFDB/lmfdb#3862]
The lmfdb_page function in Sage for an Number field doesn't work out of an ipython notebook or console for me right now (as per https://trac.sagemath.org/ticket/29126). I'm not sure who changed, sage or our searching by natural but I'm putting a note here till I have a chance to investigate (if anyone else wants to instead go ahead though!)

```
sage: E = QuadraticField(-1)
sage: E.lmfdb_page()  # optional -- webbrowser
```

**Comment by jwj61** [comment]
There was a change to normalize the "jump" box to make them work the same across different areas of the lmfdb.  I think natural was gotten rid of at that time.

**Comment by alexjbest** [comment]
@jwj61 ahh great, thanks! I see that jumping by a polynomial still works, so I'll adapt the Sage code to use whatever mechanism this happens via instead.

**Comment by JohnCremona** [comment]
Works for me from Sage command line (version 9.1) but not with 9.0.

John

**Comment by alexjbest** [comment]
I made https://trac.sagemath.org/ticket/29816 to fix this.

I think it was only merged into 9.1 so that makes sense, I don't see understand how it works for anyone right now though, as like jwj61 says the natural argument was changed to jump.

To be clear this only affected number fields, not elliptic curves.

**Comment by alexjbest** [comment]
This was merged into sage 9.2, just released so closing this.
